Micro16C.Web/src/Micro16C.Web.Client/Micro16C.Web.Client.fsproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Main.fs" />
+    <Compile Include="Startup.fs" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Bolero" Version="0.*" />
+    <PackageReference Include="Bolero.Build" Version="0.*" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="5.0.*" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\Micro16C.CompileCode\Micro16C.CompileCode.fsproj" />
```

Comment:
Did you accidently forget to commit the `Micro16C.CompileCode` project? It does not seem to be in this current revision

Reply:
I see, apparently Visual Studio, in its infinite wisdom, created the project in the folder above the solution. Should be fixed now.